scalableminds/webknossos
# Fix occluded login field

This PR fixes a bug that prevented one from focusing a login field when being prompted to login with the "does not exist or could not be accessed. You may need to log in.” toast after trying to open a dataset without being logged in.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- log out / don't log in
- open a dataset (view mode)
- you should be prompted to log in and be able to do so

### Issues:
- fixes #5521

------
